crates/db_queries/src/lib.rs

```diff
@@ -87,6 +87,15 @@ pub trait Bannable<Form> {
     Self: Sized;
 }
 
+pub trait Blockable<Form> {
```

Comment:
Please use `type Form` inside the struct instead of a generic param. I have already rewritten the other traits in the same way to simplify them, but didnt push that yet.

Reply:
Will do shortly.